health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/AddDeliverycontext.js

```
@@ -222,15 +252,29 @@
           <CardLabel className="card-label-smaller">{t(`CAMPAIGN_VALUE_LABEL`)}</CardLabel>
-          <div className="field" style={{ display: "flex", width: "100%", marginBottom: attribute?.attribute?.code === "Gender" ? null : "24px" }}>
-            {attribute?.attribute?.code === "Gender" ? (
+          <div
+            className="field"
+            style={{
+              display: "flex",
+              width: "100%",
+              marginBottom:
+                (typeof attribute?.value === "string" && /^[a-zA-Z]+$/.test(attribute?.value)) ||
+                attribute?.attribute?.valuesSchema ||
+                isNaN(attribute?.value)
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)